docs/readme.md

```diff
@@ -0,0 +1,21 @@
+## Documentation
+
+[Getting Started](docs/getting_started.md)
+
+[CRUD](docs/crud.md)
+
+[Querying](docs/querying.md)
+
+[Relationships](docs/relationships.md)
+
+[Validation](docs/validations.md)
+
+[Callbacks](docs/callbacks.md)
+
+[Migrations](docs/migrations.md)
+
+[Imports](docs/imports.md)
+
+[JSON Support](docs/json_support.md)
+
+[YAML Support](docs/yaml_support.md)
```

Comment:
All these will need to have `docs/` removed since they are in same directly as the files now.